doc/nrf/libraries/emds/emds.rst

```diff
@@ -0,0 +1,52 @@
+.. _emds_readme:
```

Comment:
I'd suggest to change `memory` on `entry` (with clarification of what entry is) in all places when entry is assumed. Currently, it is difficult to differentiate which memory the document is talking about.